re-branding-the-default-login-page/authenticationendpoint/css/travelocity.css

```diff
@@ -0,0 +1,288 @@
+/*
+ * Copyright (c) 2022, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+body {
+    perspective: 800px;
+    height: 100vh;
+    margin: 0;
+    overflow: hidden;
+    font-family: "Gudea", sans-serif;
+    background: #EA5C54;
+    /* Old browsers */
+    /* FF3.6+ */
+    /* Chrome,Safari4+ */
+    /* Chrome10+,Safari5.1+ */
+    /* Opera 11.10+ */
+    /* IE10+ */
+    background: linear-gradient(135deg, #EA5C54 0%, #bb6dec 100%);
+    /* W3C */
+    filter: progid:DXImageTransform.Microsoft.gradient(startColorstr="#EA5C54 ", endColorstr="#bb6dec", GradientType=1);
+    /* IE6-9 fallback on horizontal gradient */
+}
+
+body ::-webkit-input-placeholder {
+    color: #4E546D;
+}
+
+body .authent {
+    display: none;
+    background: #35394a;
+    /* Old browsers */
+    /* FF3.6+ */
+    /* Chrome,Safari4+ */
+    /* Chrome10+,Safari5.1+ */
+    /* Opera 11.10+ */
+    /* IE10+ */
+    background: linear-gradient(45deg, #35394a 0%, #1f222e 100%);
+    /* W3C */
+    filter: progid:DXImageTransform.Microsoft.gradient(startColorstr="#35394a", endColorstr="#1f222e", GradientType=1);
+    /* IE6-9 fallback on horizontal gradient */
+    position: absolute;
+    left: 0;
+    right: 90px;
+    margin: auto;
+    width: 100px;
+    color: white;
+    text-transform: uppercase;
+    letter-spacing: 1px;
+    text-align: center;
+    padding: 20px 70px;
+    top: 200px;
+    bottom: 0;
+    height: 70px;
+    opacity: 0;
+}
+
```

Comment:
I think it is better to be consistent with the presence or absence of the empty line after each style group. In the `avis.css` file, the empty lines are missing. Shall we either have empty lines or remove them altogether from all files?